test/ebaze_web/controllers/auction_controller_test.exs

```diff
@@ -130,11 +126,27 @@ defmodule EbazeWeb.AuctionControllerTest do
     end
   end
 
-  defp create_and_sign_in_user(_) do
-    {:ok, _user} = Accounts.create_user(%{password: "some password", username: "some username"})
+  defp create_user() do
+    {:ok, user} = Accounts.create_user(%{password: "some password", username: "some username"})
+    user
+  end
 
+  defp sign_in_user(_) do
     post(build_conn(), Routes.session_path(build_conn(), :create),
       user: %{password: "some password", username: "some username"}
     )
   end
+
+  defp auction_attrs(user) do
```

Comment:
Could these attrs be pulled out into a factory or support file so it can be shared?

Reply:
Agreed, a fixture file would be great for separation of concerns at this point.

Reply:
@ssunday would  https://github.com/thoughtbot/ex_machina be helpful ?

Reply:
@purnima-nyc Don't use a 3rd party package for this—you can make a fixture/support factory file quite easily.